From 878de6983c0f324ca6baba572b73dcba035c9c16 Mon Sep 17 00:00:00 2001
From: Harshesh Valera <harshesh.v@variscite.com>
Date: Thu, 2 Apr 2020 12:20:38 -0700
Subject: [PATCH] arm64: imx8_var_defconfig: Compile SDMA as kernel module

Use SDMA driver as a loadable kernel module, instead of part of
kernel image.

Signed-off-by: Harshesh Valera <harshesh.v@variscite.com>
---
 arch/arm64/configs/imx8_var_defconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/configs/imx8_var_defconfig b/arch/arm64/configs/imx8_var_defconfig
index e6ebfda..ed3a20b 100644
--- a/arch/arm64/configs/imx8_var_defconfig
+++ b/arch/arm64/configs/imx8_var_defconfig
@@ -525,7 +525,7 @@ CONFIG_RTC_DRV_SNVS=y
 CONFIG_RTC_DRV_IMX_SC=y
 CONFIG_DMADEVICES=y
 CONFIG_FSL_EDMA_V3=y
-CONFIG_IMX_SDMA=y
+CONFIG_IMX_SDMA=m
 CONFIG_MXS_DMA=y
 # CONFIG_MX3_IPU is not set
 CONFIG_UIO=y
-- 
2.7.4

